src/app/content/__snapshots__/utils.spec.ts.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`toRelativeUrl when at the root (unused) 1`] = `"books/book1/pages/page1"`;
+
+exports[`toRelativeUrl when deeply under the same Page (unused) 1`] = `".."`;
```

Comment:
i still don't like that this is just `..` when "toRelativeUrl when under the same Page" is `../page1`

Reply:
Since we don't use it yet can this one be removed for now? I added a `throw new Error(...)` for the cases when `path.relative(...)` returns `''`

Reply:
i mean, i also don't love introducing a constraint about being not able to link to places from certain places. 

can we check to see if the basename of the result path is  `..` and append another `..` plus the basename of the target path?